config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/metal-stack/capms-controller
+  newName: ghcr.io/metal-stack/cluster-api-metal-stack-controller
```

Comment:
If I remember correctly this change may not be committed although one script repeatedly does this change